examples/client.js

```diff
@@ -14,5 +14,7 @@ setInterval(function() {
   client.act('role:create', {
     max: 100,
     min: 25
-  }, console.log);
+  }, function(err, msg) {
```

Comment:
Would you mind not mixing stylistic changes with functional changes on the PR? :rabbit: